src/rplidar_nodelet.cpp

```diff
@@ -152,8 +155,9 @@ namespace rplidar_ros {
     start_scan_time = ros::Time::now();
     op_result = drv->grabScanData(nodes, count);
     end_scan_time = ros::Time::now();
-    scan_duration = (end_scan_time - start_scan_time).toSec() * 1e-3;
-
+    scan_duration = (end_scan_time - start_scan_time).toSec();// * 1e-3;
```

Comment:
this changes the units of the scan_duration; was this a bug or did you make a mistake here?